content/en/BestPractices/prevent-xml-external-entities-attack.md

```diff
@@ -0,0 +1,292 @@
+XML External Entities (XXE) vulnerability was ranked 4th in the OWASP Top 10 Web Application Security Risks (2017) based on its impact and likelihood, which was later merged with the Security Misconfiguration category, ranked 5th in the OWASP Top 10 2021. 
```

Comment:
```suggestion
---
title: Protect against XXE Attacks
linkTitle: Protect against XXE Attacks
weight: 680
description: >
  Securely configure your XML processor to prevent XXE attacks.
---

{{% pageinfo %}}
Learn how to protect your application from malicious Extensible Markup Language (XML) inputs.
{{% /pageinfo %}}

[OWASP](/getting-started/glossary/#open-web-application-security-project-owasp) ranked XML External Entities (XXE) vulnerability fourth in the [OWASP Top 10 Application Security Risks 2017](https://owasp.org/www-project-top-ten/2017/Top_10) based on its impact and likelihood. Later, they merged this category with Security Misconfiguration ranked fifth in the [OWASP Top 10 2021](https://owasp.org/www-project-top-ten/).
```